### PR TITLE
Add hint for public dir to hosting config

### PR DESCRIPTION
**Pull request by rabauss**
At the latest for Contao 5 you will need the config of the `public-dir`

### CODE REVIEW

File: docs/manual/installation/system-requirements.de.md [hunk 1]
```diff
@@ -229,6 +229,19 @@ automatisch anstelle von `/public` verwendet.)
 Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
 {{% /notice %}}
 
+({{< version-tag "4.13" >}} Wenn du noch den `/web`-Ordner verwendest,
```
Comment (fritzmg) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
{{< version-tag "4.13" >}} Wenn du noch den `/web`-Ordner verwendest,
```

File: docs/manual/installation/system-requirements.en.md [hunk 2]
```diff
@@ -222,6 +222,18 @@ Example: `example.com` points to the directory `/www/example/web`
 Therefore, a separate (sub)domain is required for each Contao installation.
 {{% /notice %}}
 
+({{< version-tag "4.13" >}} When your installation is still using the folder `/web` as public directory, please explicitly set it in the `composer.json`
```
Comment (fritzmg) [inline docs/manual/installation/system-requirements.en.md]:
```suggestion
{{< version-tag "4.13" >}} If your installation is still using the folder `/web` as its public directory, explicitly set it in the `composer.json`
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/manual/installation/system-requirements.en.md [hunk 3]
```diff
@@ -222,6 +222,18 @@ Example: `example.com` points to the directory `/www/example/web`
...
+of the project, to be prepared for future version of contao:
```
Comment (fritzmg) [inline docs/manual/installation/system-requirements.en.md]:
```suggestion
of the project in order to be prepared for future version of contao:
```

Comment (rabauss) [reply docs/manual/installation/system-requirements.en.md]:
```suggestion
of the project in order to be prepared for future versions of contao:
```

File: docs/manual/installation/system-requirements.en.md [hunk 4]
```diff
@@ -222,6 +222,18 @@ Example: `example.com` points to the directory `/www/example/web`
...
+of the project in order to be prepared for future versions of contao:
+
+```json
+{
+  "extra": {
+    "public-dir": "web"
+  }
+}
+```
+see also: https://symfony.com/doc/current/configuration/override_dir_structure.html#override-the-public-directory
```
Comment (fritzmg) [inline docs/manual/installation/system-requirements.en.md:236]:
```suggestion

see also: https://symfony.com/doc/current/configuration/override_dir_structure.html#override-the-public-directory
```

**Review by fritzmg — APPROVED** [review]

**Review by netzarbeiter — APPROVED** [review]
Thank you, Ralf

**Pull request merged**